metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/interfaces/MessageParser.java

```diff
@@ -34,15 +34,15 @@
    * @param rawMessage
    * @return If null is returned, this is treated as an empty list.
    */
-  List<T> parse(byte[] rawMessage);
+  List<T> parse(byte[] rawMessage, SensorParserConfig sensorParserConfig);
```

Comment:
I like this idea but I'm a little confused on the implementation.  Wouldn't just adding this do the trick:  "BasicParser would provide an overridable implementation of the two-argument form as a trivial call to the one-argument form"?  Why would we want both methods in the interface?  Am I missing something?